service/vspo-schedule/web/src/components/Elements/Button/ThemeToggleButton.tsx

```diff
@@ -76,6 +76,10 @@ export const ThemeToggleButton: React.FC = () => {
   return (
     <FormGroup>
       <FormControlLabel
+        style={{
+          marginRight: 0,
+          padding: '6px 24px',
+        }}
```

Comment:
Regarding MUI components, please use sx props instead of inline styles🙏.
https://mui.com/system/getting-started/the-sx-prop/#theme-aware-properties

(Sorry for the parts that already use style. I will create a separate issue to replace them.)

Reply:
👌 ! Should I also use **aliases** for `spacing` property? (ref: https://mui.com/system/getting-started/the-sx-prop/#spacing)
For example:
```diff
- marginRight: 0;
- padding: "6px 24px";
+ mr: 0;
+ p: 0;
```

Reply:
Yes! I would like to actively use aliases for spacing properties if possible.

Reply:
No problem 👍 Just update a new commit 87b7c4867b988f16cca2c1449c244c4fc9adadbf for this PR. Have a look!